frontend/src/components/AppFooter.js

```diff
@@ -1,18 +1,19 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import "../styles/AppFooter.css";
+import { FaTachometerAlt, FaChartBar, FaSignOutAlt } from 'react-icons/fa';
```

Comment:
When adding a new dependency, you need to add it to our package.json as well, now our app doesn't run since npm install does not know to install react-icons. You can add the dependencies to package.json during package install by running:
```
$ npm install --save react-icons
```

Reply:
Resolved by 18c6683

Reply:
Done as your suggestion. Thank you.